### DISCUSSION TITLE
Sending Login credentials from a login form from my website

### DISCUSSION THREAD

**Discussion by noelkaiye** [Intermesh/groupoffice#990] [General]
Hello there,

First of all I am not a programmer. As I was checking a number of softwares on Softaculous in my cPanel hosting account, I stumbled upon Group-Office under ERP category. I decided to install in my website and tested all features of the software. It looked great and I loved Group Office. I was trying to find a way where I can send login credentials from a form created by myself on my site to directly login to Group Office dashboard and when I logout it should return me to my homepage. I tried out an example given on https://www.group-office.com/wiki/API_calls but it doesn't login to Group Office Dashboard. Instead it takes me to Group Office index.php which is Group Office's login page. How can I get it past this page directly into logged in user dashboard??

Thank you.

/*THIS CODE*/
<?php
require('/path/to/GO.php');

$user='Get the username in a very secure way!';
$pass='Get the password in a very secure way!';

GO::session()->login($user, $pass);

header('Location: /url/to/groupoffice');

**Comment by mschering** [comment]
That's a very old example that no longer works with the current version. It's still possible though using the JMAP API but you also need to consider two factor auth.
We can develop a script for you that suits your needs. Please contact us for an offer.

**Reply by noelkaiye** [reply]
Thank you so much for this response. I have installed version 6.6.160 on my site. I would very much appreciate if a script can be developed for me. Let me know how much I can pay for that automation.

Thank you.

**Reply by noelkaiye** [reply]
I want two factor authentication. How can I get that?
